src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -1,5 +1,10 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+import javax.persistence.PrimaryKeyJoinColumn;
+
+@Entity
+@PrimaryKeyJoinColumn(name = "coach_id")
```

Comment:
what does this annotation stand for?

Reply:
annotations tell the JPA implementation how to join each subclass table record to the corresponding record in its direct superclass table.